notebooks/249-fastcomposer-image-generation/249-fastcomposer-image-generation.ipynb

```diff
@@ -0,0 +1,1327 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    @dataclass()
</pre><p>could it be a good idea to push this out to a yaml file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I think that in that case it is better to keep config in the notebook for clarity and to avoid extra requirements.